apps/crn-server/src/validation/research-output.validation.ts

```diff
@@ -64,6 +65,8 @@ const researchOutputPostRequestValidationSchema: JSONSchemaType<ResearchOutputPo
       },
       teams: { type: 'array', items: { type: 'string' }, minItems: 1 },
       accessInstructions: { type: 'string', nullable: true },
+      identifierType: { type: 'string', enum: researchOutputIdentifierTypes },
+      identifier: { type: 'string', nullable: true },
```

Comment:
im not so sure about this, shouldn't we be more explicit about the identifiers? like.. it should set the `doi` property when the `doi` type is selected. In fact - I don't think the BE should receive the `identifierType` at all? it's mostly for the FE anyway (if i remember correctly). 

What if we want to provide many at the some time at some point (Which we are able to do in squidex)? Also what if we want validation in the BE - at the moment it accepts any arbitrary string. 

Finally - should we be accepting any random string? after all we do have regexes for each one of them

Reply:
Sounds good to me. I'll see if using a regex is high effort